tools/db_bench_tool.cc

```diff
@@ -1821,6 +1827,8 @@ class Benchmark {
              / 1048576.0));
     fprintf(stdout, "Write rate: %" PRIu64 " bytes/second\n",
             FLAGS_benchmark_write_rate_limit);
+    fprintf(stdout, "Read rate: %" PRIu64 " bytes/second\n",
```

Comment:
from the code it looks to me that this should be ops/second instead of bytes/second